cedar-policy-core/src/parser/cst_to_ast.rs

```diff
@@ -1189,78 +1189,31 @@ impl ASTNode<Option<cst::Mult>> {
         let mult = maybe_mult?;
 
         let maybe_first = mult.initial.to_expr_or_special(errs);
-        // collect() preforms all the conversions, generating any errors
-        let more: Vec<(cst::MultOp, _)> = mult
+        let more = mult
             .extended
             .iter()
-            .filter_map(|&(op, ref i)| i.to_expr(errs).map(|e| (op, e)))
-            .collect();
+            .filter_map(|&(op, ref i)| i.to_expr(errs).map(|e| (op, e)));
 
+        let (more, new_errs): (Vec<_>, Vec<_>) = more
+            .map(|(op, expr)| match op {
+                cst::MultOp::Times => Ok(expr),
+                cst::MultOp::Divide => {
+                    Err(ParseError::ToAST("division is not supported".to_string()))
+                }
+                cst::MultOp::Mod => Err(ParseError::ToAST(
+                    "remainder/modulo is not supported".to_string(),
+                )),
+            })
+            .partition_result();
+        errs.0.extend(new_errs);
```

Comment:
```suggestion
        errs.extend(new_errs);
```
unless 2.4.x doesn't have the `Deref` impl that makes this work